Gems/NvCloth/Code/Source/Components/ClothComponent.cpp

```diff
@@ -14,6 +14,15 @@
 
 namespace NvCloth
 {
+    AZ_CVAR(
+        bool,
+        bg_enableNvClothSimulation,
```

Comment:
Could we force _bg_enableNvClothSimulation_ to false inside of the MultiplayerSystemCompoent if it's a dedicated server? Not a fan of cvars being set automatically, but I think most (all?) people running a dedicated server will want this disabled.

Reply:
Or is there a way for ClothComponent to exclude itself from server entities? We do something similar with Popcorn Fx particle components, where they simply won't be created on server.

Reply:
I removed the server target from NvCloth and renamed it to `cl_enableNvClothSimulation`